pkg/beaconproxy/mongodb.go

```diff
@@ -3,39 +3,42 @@ package beaconproxy
 import (
 	"runtime"
 
+	"github.com/activecm/rita/config"
+	"github.com/activecm/rita/database"
 	"github.com/activecm/rita/pkg/uconnproxy"
-	"github.com/activecm/rita/resources"
 	"github.com/activecm/rita/util"
+
 	"github.com/globalsign/mgo"
 	"github.com/vbauerster/mpb"
 	"github.com/vbauerster/mpb/decor"
+
+	log "github.com/sirupsen/logrus"
 )
 
 type repo struct {
-	res *resources.Resources
-	min int64
-	max int64
+	database *database.DB
```

Comment:
Nice, I like how this removes a level from a lot of the places where we use this struct. Makes things a bit cleaner